src/components/ComponentGenerator.jsx

```diff
@@ -249,8 +249,14 @@ export function getLegendComponent(config, legendItems, ignoreArray, interaction
                         })()
                 }
                 style={{
-                    title: { fontSize: 25, fill: config.style ? config.style.legendTitleColor : null },
-                    labels: { fontSize: 18, fill: config.style ? config.style.legendTextColor : null },
+                    title: {
+                        fontSize: config.style ? config.style.legendTitleSize || 25 : 25,
```

Comment:
Instead of ternary operator, use ` (config.style || config.style.legendTitleSize) || 25`

Reply:
if its not possible, put parentheses for the middle-expression in the ternary operator.